gdal/build.sh

```diff
@@ -1,12 +1,18 @@
 #!/bin/bash
 
-UNAME=$(uname -s)
-if [ ${UNAME} = 'Darwin' ]; then
-    export CFLAGS="-mmacosx-version-min=`xcrun --sdk macosx --show-sdk-version`"
-    export CXXFLAGS="$CFLAGS"
-    export LDFLAGS="$CFLAGS"
-fi
-
-bash configure --with-python --prefix=$PREFIX --with-sqlite3=$PREFIX
-make
+# http://www.michael-joost.de/gdal_install.html
+unset CC CPP CXX
+CONFIG_SHELL=/bin/bash bash configure \
```

Comment:
Please add a comment that explains why `CONFIG_SHELL` is set, with a link to the relevant issue in the gdal project